WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -82,6 +82,13 @@ public async Task StopAsync(Wallet wallet, CancellationToken cancellationToken)
 		await CommandChannel.Writer.WriteAsync(new StopCoinJoinCommand(wallet), cancellationToken).ConfigureAwait(false);
 	}
 
+	public bool TryGetWalletStatus(string walletName, [NotNullWhen(true)] out CoinJoinClientState? walletCoinjoinStatus)
```

Comment:
This is not the WalletStatus but the CoinJoinStatus or CoinjoinProcessStatus

Reply:
I changed to `GetCoinjoinClientState`